### ISSUE TITLE
Error from the net-worth endpoint with  `Trailing data`.

### ISSUE THREAD

**Issue by chevdor** [firefly-iii/firefly-iii#8425]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I don't recall running into this issue prior to 6.1.6 but I was also on an older version 5.4.x
See also #8278 

<details>
<summary>logs</summary>

```
Hi there,

Firefly III v6.1.6 ran into an error: Trailing data.

The error was of type "Carbon\Exceptions\InvalidFormatException".

The error occurred on/at: Fri, 19 Jan 2024 12:04:21 +0100.

This error occurred in file "/var/www/html/vendor/nesbot/carbon/src/Carbon/Traits/Creator.php" on line 690 with code 0.

The error was encountered by user #1, <redacted>.

The IP address related to this error is: <redacted> ([info](https://ipinfo.io/10.244.0.81/json?token=))
GET URL is: https://firefly.<redacted>/json/box/net-worth
The full stacktrace is below. If you think this is a bug in Firefly III, you can forward this message to [james@firefly-iii.org](mailto:james@firefly-iii.org?subject=BUG!). This can help fix the bug you just encountered.

If you prefer, you can also open a new issue on [GitHub](https://github.com/firefly-iii/firefly-iii/issues).

The full stacktrace is below:

#0 /var/www/html/vendor/nesbot/carbon/src/Carbon/Traits/Creator.php(713): Carbon\Carbon::rawCreateFromFormat()
#1 /var/www/html/app/Support/Steam.php(432): Carbon\Carbon::createFromFormat()
#2 /var/www/html/app/Support/Steam.php(524): FireflyIII\Support\Steam->balanceConverted()
#3 /var/www/html/app/Helpers/Report/NetWorth.php(97): FireflyIII\Support\Steam->balancesByAccountsConverted()

#38 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(119): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#39 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(805): Illuminate\Pipeline\Pipeline->then()
#40 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(784): Illuminate\Routing\Router->runRouteWithinStack()
#41 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(748): Illuminate\Routing\Router->runRoute()
#42 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(737): Illuminate\Routing\Router->dispatchToRoute()
#43 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(200): Illuminate\Routing\Router->dispatch()
#44 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(144): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#45 /var/www/html/app/Http/Middleware/InstallationId.php(47): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#46 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\Http\Middleware\InstallationId->handle()
#47 /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#48 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\Http\Middleware\TrustProxies->handle()
#49 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#50 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\Foundation\Http\Middleware\TransformsRequest->handle()
#51 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull->handle()
#52 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#53 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\Foundation\Http\Middleware\TransformsRequest->handle()
#54 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\Foundation\Http\Middleware\TrimStrings->handle()
#55 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#56 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\Foundation\Http\Middleware\ValidatePostSize->handle()
#57 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(99): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#58 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance->handle()
#59 /var/www/html/app/Http/Middleware/SecureHeaders.php(48): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#60 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\Http\Middleware\SecureHeaders->handle()
#61 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(119): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#62 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(175): Illuminate\Pipeline\Pipeline->then()
#63 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(144): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#64 /var/www/html/public/index.php(73): Illuminate\Foundation\Http\Kernel->handle()
#65 {main}

The following headers may also be relevant:

- host: [firefly.<redacted>](http://firefly.<redacted>/)
- user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/[120.0.0.0](http://120.0.0.0/) Safari/537.36
- accept: application/json, text/javascript, */*; q=0.01
- accept-encoding: gzip, deflate, br
- accept-language: en-US,en;q=0.9,de;q=0.8,en-DE;q=0.7,en-IE;q=0.6,de-DE;q=0.5,fr-FR;q=0.4,fr;q=0.3,en-GB;q=0.2
- sec-ch-ua: "Not_A Brand";v="8", "Chromium";v="120", "Brave";v="120"
- sec-ch-ua-mobile: ?0
- sec-ch-ua-platform: "macOS"
- sec-fetch-dest: empty
- sec-fetch-mode: cors
- sec-fetch-site: same-origin
- sec-gpc: 1
- x-csrf-token: DrFl3H2FOV5qm5OvRHL7XOvGAieyQzbWhrN2qpmH
- x-forwarded-for: 10.244.0.81
- x-forwarded-host: [firefly.<redacted>](http://firefly.<redacted>/)
- x-forwarded-port: 443
- x-forwarded-proto: https
- x-forwarded-server: traefik-6cb45f49d9-c9kt6
- x-real-ip: 10.244.0.81
- x-requested-with: XMLHttpRequest
Beep boop,

The Firefly III Mail Robot
```

</details>

### Debug information

Debug information generated at 2024-01-19 12:11:20 for Firefly III version **v6.1.6**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>v6.1.6 / v2.0.12 / 22 (exp. 22)</td></tr><tr><td>PHP version</td><td>8.3.1 / apache2handler / Linux</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>67108864 (64 MB)</td></tr><tr><td>Database drivers</td><td>mysql, pgsql, <strong>*sqlite*</strong>, </td></tr><tr><td>Docker build</td><td><span>#</span>901, base <span>#</span>68</td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Berlin + Europe/Berlin</td></tr><tr><td>App environment</td><td>local, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>info, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td>**</td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>smtp</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td><span title="Has piggy banks">:pig:</span> <span title="Has rules">:wrench:</span> <span title="Has recurring transactions">:clock130:</span> <span title="Has object groups">:bookmark_tabs:</span> <span title="Has subscriptions">:email:</span></td></tr><tr><td>Session start</td><td>2024-01-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-01-31 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36</td></tr></tbody></table>

### Expected behaviour

no error, no email, net-worth shows up

### Steps to reproduce

The issue occurs cimply by visiting the dasboard

**Comment by chevdor** [comment]
Seeing `Carbon\Exceptions\InvalidFormatException`, I am assuming there may be a date or number format issue somewhere. What is the best way to spot what piece of data is affected ?
Visiting https://firefly.redacted/json/box/net-worth shows a generic error:
```
Whoops! An error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

Trailing data
```

**Comment by JC5** [comment]
Specifically, I am curious about the stack trace that leads up to the error 😁

**Comment by chevdor** [comment]
I was curious about that too :) here it is.
I get cli and audit logs. The audit logs shows nothing interesting.

<details>
<summary>cli logs</summary>

```
[2024-01-19 12:49:32] local.DEBUG: Now in Handler::render()
[2024-01-19 12:49:32] local.DEBUG: Error "Illuminate\Auth\AuthenticationException" has no Firefly III treatment, parent will handle.
[2024-01-19 12:49:51] local.DEBUG: Found 21 accounts.
[2024-01-19 12:49:51] local.DEBUG: Will not include "<redacted>" in net worth charts.
[2024-01-19 12:49:51] local.DEBUG: Now in byAccounts("68,295,115,329,121,113,117,119,123,127,122,249,266,267,129,131,133,253,1,3", "2024-01-19")
[2024-01-19 12:49:51] local.DEBUG: Created new ExchangeRateConverter in FireflyIII\Helpers\Report\NetWorth::byAccounts
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #68, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #295, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #115, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #329, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #121, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #113, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #117, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #119, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #123, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #127, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #122, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: No conversion necessary!
[2024-01-19 12:49:51] local.DEBUG: Now in balanceConverted (2024-01-19) for account #249, converting to EUR
[2024-01-19 12:49:51] local.DEBUG: 1 transaction(s) in set #1
[2024-01-19 12:49:51] local.DEBUG: 0 transaction(s) in set #2
[2024-01-19 12:49:51] local.DEBUG: 0 transactions in set #3
[2024-01-19 12:49:51] local.DEBUG: 0 transactions in set #4
[2024-01-19 12:49:51] local.DEBUG: 0 transactions in set #5
[2024-01-19 12:49:51] local.DEBUG: 0 transactions in set #6
[2024-01-19 12:49:51] local.DEBUG: Balance from set #2 and #4 is 0.000000
[2024-01-19 12:49:51] local.DEBUG: Created new ExchangeRateConverter in FireflyIII\Support\Steam::balanceConverted
[2024-01-19 12:49:51] local.ERROR: Exception is: {"class":"Carbon\\Exceptions\\InvalidFormatException","errorMessage":"Trailing data","time":"Fri, 19 Jan 2024 12:49:51 +0100","file":"\/var\/www\/html\/vendor\/nesbot\/carbon\/src\/Carbon\/Traits\/Creator.php","line":690,"code":0,"version":"6.1.6","url":"https:\/\/firefly.k8s.chevdor.cc\/json\/box\/net-worth","userAgent":"Mozilla\/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/120.0.0.0 Safari\/537.36","json":true,"method":"GET","post":""}
[2024-01-19 12:49:52] local.ERROR: Trailing data {"userId":1,"exception":"[object] (Carbon\\Exceptions\\InvalidFormatException(code: 0): Trailing data at /var/www/html/vendor/nesbot/carbon/src/Carbon/Traits/Creator.php:690)
[stacktrace]
#0 /var/www/html/vendor/nesbot/carbon/src/Carbon/Traits/Creator.php(713): Carbon\\Carbon::rawCreateFromFormat()
#1 /var/www/html/app/Support/Steam.php(432): Carbon\\Carbon::createFromFormat()
#2 /var/www/html/app/Support/Steam.php(524): FireflyIII\\Support\\Steam->balanceConverted()
#3 /var/www/html/app/Helpers/Report/NetWorth.php(97): FireflyIII\\Support\\Steam->balancesByAccountsConverted()
#4 /var/www/html/app/Http/Controllers/Json/BoxController.php(274): FireflyIII\\Helpers\\Report\\NetWorth->byAccounts()
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): FireflyIII\\Http\\Controllers\\Json\\BoxController->netWorth()
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(43): Illuminate\\Routing\\Controller->callAction()
#7 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(259): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#8 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\\Routing\\Route->runController()
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(806): Illuminate\\Routing\\Route->run()
#10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(144): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#11 /var/www/html/app/Http/Controllers/Controller.php(118): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(165): FireflyIII\\Http\\Controllers\\Controller->FireflyIII\\Http\\Controllers\\{closure}()
#13 /var/www/html/app/Http/Middleware/InterestingMessage.php(71): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\\Http\\Middleware\\InterestingMessage->handle()
#15 /var/www/html/app/Http/Middleware/Binder.php(75): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#16 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\\Http\\Middleware\\Binder->handle()
#17 /var/www/html/app/Http/Middleware/Range.php(56): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#18 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\\Http\\Middleware\\Range->handle()
#19 /var/www/html/vendor/jc5/google2fa-laravel/src/Middleware.php(29): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#20 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): PragmaRX\\Google2FALaravel\\Middleware->handle()
#21 /var/www/html/app/Http/Middleware/Authenticate.php(66): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#22 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\\Http\\Middleware\\Authenticate->handle()
#23 /var/www/html/vendor/laravel/passport/src/Http/Middleware/CreateFreshApiToken.php(63): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#24 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Laravel\\Passport\\Http\\Middleware\\CreateFreshApiToken->handle()
#25 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/AuthenticateSession.php(60): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#26 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Session\\Middleware\\AuthenticateSession->handle()
#27 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#28 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
#29 /var/www/html/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#30 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
#31 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#32 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
#33 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Session\\Middleware\\StartSession->handle()
#34 /var/www/html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#35 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#36 /var/www/html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#37 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#38 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(119): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#39 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(805): Illuminate\\Pipeline\\Pipeline->then()
#40 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(784): Illuminate\\Routing\\Router->runRouteWithinStack()
#41 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(748): Illuminate\\Routing\\Router->runRoute()
#42 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(737): Illuminate\\Routing\\Router->dispatchToRoute()
#43 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(200): Illuminate\\Routing\\Router->dispatch()
#44 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(144): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#45 /var/www/html/app/Http/Middleware/InstallationId.php(47): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#46 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\\Http\\Middleware\\InstallationId->handle()
#47 /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#48 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Http\\Middleware\\TrustProxies->handle()
#49 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#50 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#51 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
#52 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#53 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#54 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
#55 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#56 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#57 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(99): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#58 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#59 /var/www/html/app/Http/Middleware/SecureHeaders.php(48): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#60 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): FireflyIII\\Http\\Middleware\\SecureHeaders->handle()
#61 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(119): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#62 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(175): Illuminate\\Pipeline\\Pipeline->then()
#63 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(144): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#64 /var/www/html/public/index.php(73): Illuminate\\Foundation\\Http\\Kernel->handle()
#65 {main}
"}
[2024-01-19 12:49:52] local.DEBUG: Now in Handler::render()
[2024-01-19 12:49:52] local.DEBUG: Error "Carbon\Exceptions\InvalidFormatException" has no Firefly III treatment, parent will handle.
```

</details>

**Comment by chevdor** [comment]
My account #249 btw is an account using a custom currency (`DAI`).
There may be an issue fetching a rate for that one. I tried excluding the account from the net worth summary but that did not fix the issue.

**Comment by JC5** [comment]
No that's not the issue, it's just one of the many qualities of SQLite. I've found the issue and will push a fix for the next release.

**Comment by chevdor** [comment]
Nice, I see a lot of refactoring in the fix but I spotted [this change](https://github.com/firefly-iii/firefly-iii/commit/122f0309a65f9a6744290746c3a36a652e958a64#diff-b0dad0b0c0a4fea6853979ddf40dc1fad6d013c51c91c24a4e9c01e04551e1dfR433-R439). 

And yes indeed, I am using sqlite and plan to stick with it since I realized it is wayyy faster than postgres/mysql with way lower resources. In my (single user) case, I think this choice makes sense. Since I Use sqlite, it is more snappier. Backups are also much easier to do.

Thanks for the quick fix !
